js/src/modal/modal.spec.js

```diff
@@ -602,6 +632,1493 @@ describe('Modal', () => {
 
       modal.show()
     })
+
+    it('should not add compensation padding when body boundary width is less than window width with no vertical scrollbar before showing and no vertical scrollbar after showing', done => {
+      const dist = document.createElement('link')
+      dist.rel = 'stylesheet'
+      dist.id = 'bootstrap-dist'
+      dist.href = '/base/dist/css/bootstrap.css'
+      document.head.appendChild(dist)
+
+      document.body.style.width = window.innerWidth - 1 + 'px'
+      document.body.style.height = '1px'
+      document.documentElement.style.paddingRight = null
+      fixtureEl.style.position = 'relative'
```

Comment:
If you change `fixtureEl` style please applied the default style after each of your unit tests